### DISCUSSION TITLE
Blockquote

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#219] [Component suggestions]
<img width="480" alt="visuele weergave van het accordion component" src="https://github.com/nl-design-system/backlog/assets/545207/7731a9a6-208d-43be-911b-a97273b72251">

We willen graag meer leren over het **Blockquote** component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook of screenshots. Met deze informatie kunnen we bewijzen dat het component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1708010183644919)

---

## Naam
Blockquote

## Doel
Toont een citaat dat afwijkt van de hoofdtekst.

## Gerelateerde componenten
[Rich Text Content](https://github.com/orgs/nl-design-system/discussions/289)

## Onderzoek
Niet veel onderzoek kunnen vinden. Enkel deze [comment in de 'Pull quote' Github discussion van gov.uk](https://github.com/alphagov/govuk-design-system-backlog/issues/258#issuecomment-1216541086).

**Comment by jeffreylauwers** [comment]
## Amsterdam

- [Storybook](https://designsystem.amsterdam/?path=/docs/components-text-blockquote--docs)
- [GitHub](https://github.com/Amsterdam/design-system/blob/develop/packages/css/src/components/blockquote)
- [Figma](https://www.figma.com/file/9IGm6IdPUYizBNGsUnueBd/Amsterdam-Design-System?type=design&node-id=9734-83744&mode=design&t=6jEcP4ZnEHLWoh74-4)

---

### Screenshots / Varianten
**Storybook**
![Scherm­afbeelding 2024-02-19 om 13 22 09](https://github.com/nl-design-system/backlog/assets/545207/52edf11a-b914-4d52-be59-3a09d7138945)

Inverse colour
![Scherm­afbeelding 2024-02-19 om 13 22 15](https://github.com/nl-design-system/backlog/assets/545207/45c947d0-2ce6-4693-a082-61cbf22791cb)

**Figma**
![Scherm­afbeelding 2024-02-19 om 13 24 04](https://github.com/nl-design-system/backlog/assets/545207/7729d557-2ce9-4ddb-9e03-01e467b97fcb)

---

### Notities
- Storybook component biedt de mogelijkheid om de kleur om te keren bij een donkere achtergrond.
- Richtlijnen beschikbaar.
- Figma component biedt de mogelijkheid voor attribution.

**Comment by jeffreylauwers** [comment]
## Den Haag

- [Storybook](https://nl-design-system.github.io/denhaag/?path=/docs/css-data-display-blockquote--docs)
- [Figma - Huidig](https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn/%F0%9F%8C%BB-Den-Haag-Design-System?type=design&node-id=8123-22417&mode=design&t=ZLuBqxc9PtvKHvhr-4)
- [Figma - WIP](https://www.figma.com/file/x4RkF6BIdrNZbh7D53NTzB/%F0%9F%92%A0-NLDS---Den-Haag---Bibliotheek?type=design&node-id=1349-6341&mode=design&t=nONFbSB0oTetl61B-4)
- [Github](https://github.com/nl-design-system/denhaag/blob/main/components/Blockquote/src/index.scss)

---

### Screenshots / Varianten

**Storybook**

Default
![Scherm­afbeelding 2024-02-19 om 13 30 37](https://github.com/nl-design-system/backlog/assets/545207/8acd4cb1-a036-4540-b3c7-0929d959cca6)

With attribution
![Scherm­afbeelding 2024-02-19 om 13 30 42](https://github.com/nl-design-system/backlog/assets/545207/1aaa049f-70ac-4d5d-b48e-e7902b71f45c)

---


### Notities
-  Richtlijnen beschikbaar.
- Border inline-start.

**Comment by jeffreylauwers** [comment]
## DUO

[Documentatie website](https://uno.dfront.rijkscloud.nl/#/components/quote)

---

### Screenshots / Varianten

**Documentatie website**

Default
![Scherm­afbeelding 2024-02-19 om 13 38 40](https://github.com/nl-design-system/backlog/assets/545207/5a56db8f-6393-44d4-956a-e9c5e5edb68e)

Met afbeelding
Gebruik een quote met afbeelding, als je bij een uitspraak wat speelsheid / visualisatie wil toevoegen. Alleen foto‘s die de gequote persoon afbeelden zijn toegestaan.
![Scherm­afbeelding 2024-02-19 om 13 38 56](https://github.com/nl-design-system/backlog/assets/545207/c4c19cc7-cc95-4063-8063-27b6904b5b67)


---

### Notities
- Onder de naam: Quote.
- Richtlijnen beschikbaar.
- Optie voor een afbeelding.

**Comment by jeffreylauwers** [comment]
## Utrecht

- [Storybook](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css-blockquote--docs)
- [GitHub](https://github.com/nl-design-system/utrecht/tree/main/components/blockquote)
- [Figma](https://www.figma.com/file/UXIHcIurAD8hyoBWx4hDBV/NLDS---Gemeente-Utrecht---Bibliotheek?type=design&node-id=1349-6341&mode=design&t=M6QfA7La6pGFbbEl-4)

---

### Screenshots / Varianten

**Storybook**

Default
![Scherm­afbeelding 2024-02-19 om 13 57 19](https://github.com/nl-design-system/backlog/assets/545207/49eea2f6-0c4c-437b-a25e-a9244a6b7b9a)

With attribution
![Scherm­afbeelding 2024-02-19 om 13 57 30](https://github.com/nl-design-system/backlog/assets/545207/b350b5f5-5ec7-401e-b2f0-25883fc22e6b)

Distanced
![Scherm­afbeelding 2024-02-19 om 13 57 45](https://github.com/nl-design-system/backlog/assets/545207/e4fc278f-e4e5-4308-b2e4-0dc8dd23d9e9)

In another language
![Scherm­afbeelding 2024-02-19 om 13 58 12](https://github.com/nl-design-system/backlog/assets/545207/006ae15b-a1cf-4ba0-9670-386a7b8f8033)

**Figma**

![Scherm­afbeelding 2024-02-19 om 16 11 10](https://github.com/nl-design-system/backlog/assets/545207/a31ab02a-fb87-4329-acf5-1b29ac24e060)

---

### Notities
- Verschil tussen Figma en Storybook. Kleur rood vs donkergrijs. Attribution in Figma een andere kleur als quote.

**Comment by jeffreylauwers** [comment]
## RVO

[Demo pagina - Storybook](https://nl-design-system.github.io/rvo/?path=/story/demo-pagina-s-algemeen-content--default)

---

### Screenshots / Varianten

**Storybook**

![Scherm­afbeelding 2024-02-26 om 11 59 29](https://github.com/nl-design-system/backlog/assets/545207/bb32ca8f-96a5-451c-ab88-78d686804762)


---

### Notities
- Niet als los component, maar bij toeval gevonden op een demo pagina.

**Comment by jeffreylauwers** [comment]
## ✨ Dit component is nu Help Wanted ✨

Help je mee hem door de Community stappen te krijgen?
[Blockquote op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=blockquote)

**Comment by jeffreylauwers** [comment]
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[Blockquote op het Candidate bord](https://github.com/orgs/nl-design-system/projects/32/views/1?filterQuery=blockquote)

**Comment by MarjonBakker** [comment]
Ik heb net het artikel van Heydon Pickering zitten lezen: [The blockquote element](https://heydonworks.com/article/the-blockquote-element/).

Een `blockquote` hoort volgens de HTML-specificatie alleen gebruikt te worden voor quotes uit een externe bron. Maar een veelgeziene use case is de 'pull quote', waarin een citaat van een geïnterviewde wordt uitgelicht om extra aandacht te trekken. Dit is een erfenis uit kranten-/tijdschriften, waar het nodig was om de aandacht te trekken, zodat de lezer stopt met bladeren en begint met lezen. Dit heeft online een andere functie gekregen. Het breekt de pagina op en maakt hem visueel aantrekkelijker.

Voor toegankelijkheid is het een lastige. Aan de ene kant wil je semantische HTML gebruiken, maar bij een pull quote is een `blockquote` niet gepast. Als je niet-semantische HTML gebruikt, wordt voor een screenreadergebruiker een stukje content gedubbeld, zonder dat duidelijk is hoezo en waarom. Dan is dat een fout onder 1.3.1 Info en relaties.

Wat te doen? Pickering zou zelf voor pull quotes een `aside` gebruiken, in combinatie met `arialabelledby`.

Maar wat is handig voor NL Design System?
